task/kind/0.1/README.md

```diff
@@ -0,0 +1,26 @@
+# kind
+
+This configures a kind (Kubernetes-in-Docker) environment to be ran in a Task.
+
+See https://kind.sigs.k8s.io for more details.
```

Comment:
In some tasks we specify a `kubectl` command to install the task.
We could also to promote instead a `tkn` command to do that, but maybe that's something the hub could show instead of having it in all READMEs?